### PR TITLE
Updated the German CL with new CL template

### PR DESCRIPTION
**Pull request by adeclosset**
To be merged only after tomorrow's chat.

FYI - @greysteil @jutta-frieden

### CODE REVIEW

**Comment by adeclosset** [comment]
@greysteil - want to check & merge?

FYI - @jesssummerfield @gabrielamatias

File: app/pages/about/jobs/positions/country-lead-de-at.js [hunk 1]
```diff
@@ -9,85 +9,90 @@ export default class CountryLeadGermanyAustria extends React.Component {
...
         </p>
         <p className='para'>
-          We've built a world-class product that brings cutting-edge recurring payments
-          technology to a wide variety of businesses, from sole traders to multinational
-          corporations like TripAdvisor, The Guardian and Box. We've gained great
-          traction in the UK, and we know we can make recurring payments simpler for
-          everyone in the Eurozone &ndash; and beyond.
+          We've built a world-class product that brings cutting-edge recurring
+          payments technology to a wide variety of businesses, from SMEs to
+          multinational corporations like Tripadvisor, Habitat and Box.
```
Comment (greysteil) [inline app/pages/about/jobs/positions/country-lead-de-at.js]: `TripAdvisor`

File: app/pages/about/jobs/positions/country-lead-de-at.js [hunk 2]
```diff
@@ -9,85 +9,90 @@ export default class CountryLeadGermanyAustria extends React.Component {
...
+        <ul className='list'>
+          <li>
+            Sales: identifying prospective customers and successfully closing
+            them.
+          </li>
+          <li>
+            Marketing & PR: building the brand in the market.
+          </li>
+          <li>
+            Customer success: helping your customers go live and scale up
```
Comment (greysteil) [inline app/pages/about/jobs/positions/country-lead-de-at.js]: Missing a full stop.

File: app/pages/about/jobs/positions/country-lead-de-at.js [hunk 3]
```diff
@@ -9,85 +9,90 @@ export default class CountryLeadGermanyAustria extends React.Component {
...
+        to be adept at discussing complex situations with a wide variety of
+        audiences. You'll feed your market's needs into our development roadmap
+        to ensure we're always building the product people want.
         </p>
         <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>
           The ideal candidate
         </h4>
+        <ul className='list u-margin-Tm'>
+          <li>
+            You’re driven and adaptable, able and willing to take on a variety
```
Comment (greysteil) [inline app/pages/about/jobs/positions/country-lead-de-at.js]: Replace the apostrophe with a normal one (this one looks like a special character).

File: app/pages/about/jobs/positions/country-lead-de-at.js [hunk 4]
```diff
@@ -9,85 +9,90 @@ export default class CountryLeadGermanyAustria extends React.Component {
...
+          <li>
+            You have excellent written and oral communication skills,
+            and can adapt to your audience.
+          </li>
+          <li>
+            You are keen to understand technology at a granular level.
+          </li>
+          <li>
+            You're a native-level German speaker with experience
+            working in Germany or Austria and fluent in English.
```
Comment (greysteil) [inline app/pages/about/jobs/positions/country-lead-de-at.js]: `and are fluent...'

File: app/pages/about/jobs/positions/country-lead-de-at.js [hunk 5]
```diff
@@ -9,85 +9,90 @@ export default class CountryLeadGermanyAustria extends React.Component {
...
-        </p>
-        <p className='para'>
-          You're adaptable. You can tailor our voice to local conditions and needs,
-          whether you're working with prospective clients or internal teams.
-        </p>
-        <p className='para'>
-          You're a native-level German speaker with experience working in Germany,
-          Austria, or both.
+          As part of your application, please describe how you would launch
+          GoCardless in the country in three paragraphs.
```
Comment (greysteil) [inline app/pages/about/jobs/positions/country-lead-de-at.js]: `the country` -> `Germany`

File: app/pages/about/jobs/positions/country-lead-de-at.js [hunk 6]
```diff
@@ -9,85 +9,90 @@ export default class CountryLeadGermanyAustria extends React.Component {
...
         </p>
         <a href='http://boards.greenhouse.io/gocardless/jobs/52965' className='btn u-margin-Vs'>
           Apply now
         </a>
         <p className='para'>
-          Industry competitive salary. You will be based in London with travel
-          to the market.
+          Competitive salary. You will be based in London with travel to the
+          market. We will contribute towards international relocation costs
```
Comment (greysteil) [inline app/pages/about/jobs/positions/country-lead-de-at.js]: `the market` -> `Germany and Austria`.

**Comment by greysteil** [comment]
Looks awesome. Big :+1: after the above typos.

**Pull request merged**